### DISCUSSION TITLE
Firmware update  issue (v1.6.0)

### DISCUSSION THREAD

**Discussion by jonydebu** [gicamm/homeassistant-comelit#19] [General]
Hi gicamm shortly comelit update de 20003150 firmware v1.6.0 . 
Basically now hubs change port to 443 instead of 80 and add password to get in in config url , in my case 192.168.1.252:8080 .
Since your addon stop working.

Apreciate any kind of help.

Thanks in advance

**Comment by gicamm** [comment]
Hi jony,
So they changed these parameters?
`    port: 1883
    mqtt-user: hsrv-user
    mqtt-password: sf1nE9bjPc
`
Any knowledge about the new parameters?

**Comment by jonydebu** [comment]
![image](https://user-images.githubusercontent.com/30730098/208193572-2d3be625-2cb2-4290-881f-08732ddf2e7a.png)

**I think it´s easier for you in Italian. That´s what the update talk about.
Hope it helps you**.

**Comment by jonydebu** [comment]
And it continue with this ...
![image](https://user-images.githubusercontent.com/30730098/208193861-87ac55d3-4854-4854-9d2e-b87f9c71b4aa.png)

**Comment by gicamm** [comment]
You are getting confused.
Finally, they added a restriction for updating the HUB.
It was unrealistic that everyone could update the configuration without any password.
However, this means they changed the way the SimpelProg reached the HUB, and not the MQTT itself.
These are 2 different stuff.
If your APP (android or IOS) reaches the HUB and works, maybe they made some changes in the protocol or the MQTT user/pwd.
Otherwise, it would be best if you restored your configuration. 
The MQTT port is 1883. Not 443, 80, or 8080. 
Currently, I dont have any HUB with that firmware.
I could check if the protocol is the same, but I need a HUB with the new firmware. 
In any case, you should check what I wrote.
Who built your system?
